automated_testing/testcases/apply_config_from_file/run_test.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+set -xeuo pipefail
+
+hab pkg install core/redis
+hab pkg exec core/redis redis-cli --version
+
+hab svc load core/redis --remote-sup=alpha.habitat.dev
+hab svc load core/redis --remote-sup=beta.habitat.dev
+
+# TODO Wait until redis is available
```

Comment:
Nothing to change here, just lamenting.  Core-plans has the same issue in that we don't have a great way in our tooling to say 'start this service, and let me know when it's healthy' :(

Reply:
After #7148 is merged, we can convert these to pester tests and leverage `Wait-SupervisorService` that polls a service for a given timeout.